### ISSUE TITLE
If the HTML table not fit a page dompdf will break

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#301]
```
If a HTML table not fit a page dompdf will break. In my case, I have this error:

Fatal error: Maximum execution time of 60 seconds exceeded in 
C:\xampp\htdocs\dgpu8\inc\dompdf\include\style.cls.php on line 717

Version: dompdf_0-6-0_beta2
Operating system: Windows Vista

Thanks!
```

Original issue reported on code.google.com by `eana...@gmail.com` on 8 Jun 2011 at 5:21
- Merged into: #91

**Comment by GoogleCodeExporter** [comment]
```
DOMPDF should be able to flow a table normally. The problem you are 
experiencing typically happens when an individual cell within a table won't fit 
on a page.

See Issue 91.
```

Original comment by `eclecticgeek` on 8 Jun 2011 at 6:27

**Comment by GoogleCodeExporter** [comment]
```
You right, the problem was with an individual cell containig nested table.
```

Original comment by `eana...@gmail.com` on 9 Jun 2011 at 4:56

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the update. We hope to have a fix for the final 0.6.0 release.
```

Original comment by `eclecticgeek` on 9 Jun 2011 at 5:14
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:16
- Added labels: **Restrict-AddIssueComment-Commit**
